extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -267,7 +267,15 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   waitForMediaLayout_() {
-    const mediaSet = scopedQuerySelectorAll(this.element, PAGE_MEDIA_SELECTOR);
+    let mediaSet = scopedQuerySelectorAll(this.element, PAGE_MEDIA_SELECTOR);
+    const iframe = this.element.querySelector('iframe');
+    const iframeDoc = iframe && iframe.contentDocument;
+    if (iframeDoc) {
+      mediaSet = Array.from(mediaSet);
+      const fieMedia = scopedQuerySelectorAll(iframeDoc, PAGE_MEDIA_SELECTOR);
+      fieMedia.forEach(el => mediaSet.push(el));
```

Comment:
Nit: maybe using the ES6 syntax?

````
const fieMedia = scopedQuerySelectorAll(iframeDoc, PAGE_MEDIA_SELECTOR);
mediaArray = [...Array.from(mediaSet), ...fieMedia]
````

And since you're making it an array, we could replace the line below to ``mediaArray.map(...)``

Reply:
We aren’t currently allowing the spread operator 😕

Reply:
Oh... Maybe a good old ``mediaArray = Array.from(mediaSet).concat(fieMedia)`` then?

Reply:
Yeah this is also weird because you can't `concat` a `nodeList`. It just pushes the whole `nodeList` into the end of the array (I only know this because I tried it yesterday). I will change this around a bit to always return an array as suggested below anyway.

Reply:
talked offline. Changed to `concat` for readability.